pkg/metrics/statsd/metrics.go

```diff
@@ -12,11 +12,11 @@ import (
 
 // metric represents generic metric structure
 type metric struct {
-	metric    string
```

Comment:
IMHO having an attribute with the same name as the struct to which it belongs is a bit weird.